src/envs/behavior.py

```diff
@@ -30,9 +29,8 @@
 
     _BEHAVIOR_IMPORTED = True
     bddl.set_backend("iGibson")  # pylint: disable=no-member
-    if os.path.exists("tmp_behavior_states/"):
-        shutil.rmtree("tmp_behavior_states/")
-    os.makedirs("tmp_behavior_states/")
+    if not os.path.exists("tmp_behavior_states/"):
```

Comment:
with this change, our local `tmp_behavior_states/` directories are going to grow over time, and we'll have to periodically delete them. I guess that's okay, because the same can be said for `results/`, `videos/`, etc.

Reply:
Yah I don't know a better way